frontend/src/properties/propertysections/FlexboxContainerPropertySection.tsx

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { widgetToPropertySection } from 'properties/PropertySectionOperations';
+import {
+  FlexboxContainerPropertySectionProps,
+  FlexboxContainerPropertySectionStyleProps,
+} from 'properties/propertysections/FlexboxContainerPropertySection.types';
+import { PropertySectionLabel } from 'properties/propertysections/PropertySectionLabel';
+import React from 'react';
+
+const useWidgetContainerPropertySectionStyles = makeStyles<Theme, FlexboxContainerPropertySectionStyleProps>(
+  (theme) => ({
+    children: {
+      display: 'flex',
+      flexWrap: 'wrap',
```

Comment:
We don't want to set such a value in the frontend. If this value is needed, it should be configurable at least in the flexbox container description in the backend.

Reply:
Done